lib/maker.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+var _ = require('lodash');
+var async = require('async');
+var request = require('request');
+
+function init (env) {
+
+  var key = env.extendedSettings && env.extendedSettings.maker && env.extendedSettings.maker.key;
+
+  var TIME_30_MINS_MS = 30 * 60 * 1000;
+
+  function maker() {
+    return maker;
+  }
+
+  var lastAllClear = 0;
+
+  maker.sendAllClear = function sendAllClear (callback) {
+    if (Date.now() - lastAllClear > TIME_30_MINS_MS) {
+      lastAllClear = Date.now();
+      maker.makeRequest({}, 'allclear', function allClearCallback (err) {
+        if (err) {
+          lastAllClear = 0;
+          callback(err);
+        } else {
+          callback && callback(null, {sent: true});
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/jasoncalabrese/cgm-remote-monitor/file/758176594/issues/source?bid=2449627&fileBranchId=2449700#l27)